example.main.scss

```diff
@@ -2,7 +2,7 @@
    INUITCSS
    ========================================================================== */
 
-/*!*
```

Comment:
In pre-alpha, this comment was in [settings.defaults](https://github.com/inuitcss/settings.defaults/blob/master/_settings.defaults.scss) to ensure inuitcss credit is always present.

Do you think that we need to move it in [_settings.core.scss](https://github.com/inuitcss/inuitcss/blob/develop/settings/_settings.core.scss) ?